spec/tapioca/cli/gem_spec.rb

```diff
@@ -120,6 +144,32 @@ def fizz; end
           refute_path_exists("#{repo_path}/sorbet/rbi/gems/baz@0.0.2.rbi")
         end
 
+        it "must generate a gem RBI and merges exported gem RBIs" do
+          output = execute("gem", "bar")
+
+          compiling_spec = "Compiling bar, this may take a few seconds..."
+          assert_includes(output, compiling_spec)
+
+          conflict_spec = "Conflicting definitions for `::Bar::Test#foo(a, b)`"
+          assert_includes(output, conflict_spec)
+
+          assert_path_exists("#{outdir}/bar@0.3.0.rbi")
+          assert_equal(BAR_RBI, File.read("#{outdir}/bar@0.3.0.rbi"))
+        end
+
+        it "must generate a gem RBI and resolves conflicts with exported gem RBIs by keeping the generated RBI" do
+          output = execute("gem", "fizz")
+
+          compiling_spec = "Compiling fizz, this may take a few seconds..."
+          assert_includes(output, compiling_spec)
+
+          conflict_spec = "Conflicting definitions for `::Fizz::<self>#baz(a, b)`"
```

Comment:
A more precise message with [these infos](https://github.com/Shopify/rbi/blob/main/lib/rbi/loc.rb#L8-L12) would be way more useful to solve the problems.